main.js

```diff
@@ -439,7 +453,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
         });
```

Comment:
I tried that but when pushing about 1000 trading cards to the market it still got me blocked. So I added this workaround which did what I wanted. If you don't like the delay, at least add the error handling which was really important. Without it the script would stop after some trading cards.